ste/mgr-JobPartMgr.go

```diff
@@ -799,6 +799,10 @@ func (jpm *jobPartMgr) deleteSnapshotsOption() common.DeleteSnapshotsOption {
 	return jpm.Plan().DeleteSnapshotsOption
 }
 
+func (jpm *jobPartMgr) permanentDeleteOption() common.PermanentDeleteOption {
```

Comment:
I'm wondering why some functions in this file are public while others are private.